SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/SlaveMethods/StatisticsHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+using Plugin.Microsoft.Azure.SignalR.Benchmark;
+
+namespace Plugin.Microsoft.Azure.SignalR.Benchmark.SlaveMethods
+{
+    public static class StatisticsHelper
+    {
+        private static long _latencyStep = 100;
+        private static long _latencyMax = 1000;
+
+        public static void IncreaseSentMessage(ConcurrentDictionary<string, object> statistics)
+        {
+            statistics.AddOrUpdate(SignalRConstants.StatisticsMessageSent, (long)1, (k, v) => (long)v + 1);
```

Comment:
It is weird to cast object to long in this class because StatisticsHelper has no idea of the Dictionary. It is better create a counter class as before. The counter class contains Dictionary and it knows what the key and value should be.